src/plugins/workspace/public/components/workspace_menu/workspace_menu.tsx

```diff
@@ -67,68 +86,68 @@ export const WorkspaceMenu = ({ coreStart }: Props) => {
     setPopover(false);
   };
 
-  const workspaceToItem = (workspace: WorkspaceObject) => {
+  const workspaceUseCaseToItem = (
+    workspace: WorkspaceObject,
+    index: number,
+    workspaceURL: string
+  ) => {
+    const useCase = workspace.features
+      ?.map(getUseCaseFromFeatureConfig)
+      .filter(isNotNull)
+      .map((uc) => {
+        return {
+          name: uc,
+          // TODO: this should be a link to the use case overview page.
+          onClick: () => {
+            window.location.assign(workspaceURL);
+          },
+          'data-test-subj': `context-menu-item-${workspace.id}-${uc}`,
+          icon: <EuiIcon type="bolt" />,
+        };
+      });
+
+    useCasePanels.push({
+      id: index,
+      title: 'use case',
+      width: 300,
+      items: useCase,
+    });
+  };
+
+  const workspaceToItem = (workspace: WorkspaceObject, index: number) => {
     const workspaceURL = formatUrlWithWorkspaceId(
       coreStart.application.getUrlForApp(WORKSPACE_OVERVIEW_APP_ID, {
         absolute: false,
       }),
       workspace.id,
       coreStart.http.basePath
     );
-    const name =
-      currentWorkspace?.name === workspace.name ? (
-        <EuiText>
-          <strong>{workspace.name}</strong>
-        </EuiText>
-      ) : (
-        workspace.name
-      );
+
+    workspaceUseCaseToItem(workspace, index, workspaceURL);
+
+    const name = (
+      <EuiText
+        onClick={(e) => {
+          e.stopPropagation();
+          window.location.assign(workspaceURL);
+        }}
+      >
+        {workspace.name}
+      </EuiText>
```

Comment:
Why aren't we just making this a link now vs clickable text? It seems like we have everything we need, right?

Reply:
```
<EuiLink
        href={workspaceURL}
        onClick={(e) => e.stopPropagation()}
        color='text'>
   {truncate(workspace.name, { length: MAX_WORKSPACE_NAME_LENGTH })}
</EuiLink>
```
The reason I didn't use `EuiLink` is because I wanted its CSS format to be the same as the use case menu. The special handling of the `name` in the `contextmenu` is to prevent page jumping when clicking the right arrow to enter the use case menu.